wai-app-static/WaiAppStatic/Storage/Filesystem.hs

```diff
@@ -120,8 +119,8 @@ webAppLookup hashFunc prefix pieces =
 -- exists.
 hashFile :: FilePath -> IO ByteString
 hashFile fp = do
-    h <- Crypto.Hash.Conduit.hashFile fp
-    return $ B64.encode $ toBytes (h :: Digest MD5)
+    f <- BL.readFile fp
```

Comment:
This has the potential to leak FDs. A better approach would be `withBinaryFile`, which will guarantee that the FD is closed.